### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by garrett-vangilder**
## Status
**READY/IN DEVELOPMENT**

## Documentation

[X] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[ ] This PR does **NOT** require documentation.

## Description
A few sentences describing the overall goals of the pull request's commits.

This PR intends to update documentation for users.

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

n/a

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

1. n/a

## Impacted Files in Application
List general components of the application that this PR will affect:

* README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,12 +22,12 @@ This command sets the environment for your local copy of the application to deve
 export ASPNETCORE_ENVIRONMENT="Development"
 ```
 
-On initial installation of the Banagazon web application you must set an environment variable to your local database. This variable must point the application to the database
+On initial installation of the Banagazon web application you must set an environment variable to your local database. Even though you have not yet created a database, this variable will set the location of your database for later after creation.
```
Comment (daynewright) [inline README.md]: Wording seems a little weird. Maybe something like:

On initial installation of the Banagazon web application you must set an environment variable to your local database.  _(This database with the file name `bangazon.db` will be created when you run `dotnet ef database update` later in the installation process.)_

**Review by daynewright — CHANGES_REQUESTED** [review]
Wording suggestion

**Comment by garrett-vangilder** [comment]
@TeamCharles/team changes made

**Review by MattKraatz — APPROVED** [review]
LGTM

**Review by daynewright — APPROVED** [review]

**Pull request merged**